src/api/wishlist.js

```diff
@@ -0,0 +1,28 @@
+import Base from './base';
+
+export default class extends Base {
+    /**
+     * @Constructor
+     */
+    constructor(version) {
+        // call parent
+        super(version);
+
+        // set up class variables
+        this.endpoint = '/wishlist.php?action=add&wishlistid=';
+        this.productIdKey = '&product_id=';
```

Comment:
Maybe  just`productKey` ?

Reply:
maybe we can use other approach like 
`this.endpoint = '/wishlist.php?action=add&wishlistid=[WISHLIST_ID]&product_id=[PRODUCT_ID]';

const url = endpoint.replace('[WISHLIST_ID]', wishlistId).replace('[PRODUCT_ID]', productId);
`

Reply:
@yurytut1993 done!